assets/javascripts/audit_log.js

```diff
@@ -309,6 +309,35 @@ function loadProductLogTable(dataTableUrl, rescheduleUrlTemplate, showActions) {
         }
       },
       {targets: 2, orderable: false},
+      {
+        targets: 6,
+        render: function (data, type, row) {
+          const rowData = dataForLink(this);
+          const settings = rowData.settings;
+          const version = settings['VERSION'];
+          const build = settings['BUILD'];
+          const distri = settings['DISTRI'];
+          if (version !== undefined && build !== undefined && distri !== undefined) {
+            return `<a href=/tests/overview?version=${version}&build=${build}&distri=${distri}&flavor=${data}>${data}</a>`;
```

Comment:
The attribute value should be quoted. I'm also wondering whether we need to do proper escaping here. (It is very annoying that one cannot just return a DOM element here.)

The same counts for the other occurrence.